modules/standard/VisualDebug.chpl

```diff
@@ -53,25 +53,39 @@ module VisualDebug
 pragma "no doc"
   enum vis_op {v_start, v_stop, v_tag, v_pause};
 
-  private proc VDebugTree (what: vis_op, name: string, time: real, id: int = 0) {
-      var child = id * 2 + 1;
+pragma "no doc"
```

Comment:
Is there any reason for the `pragma "no doc"` instead of just making this a private iterator?